src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE_PATTERN = "dd.MM.yyyy";
+    private static final DateTimeFormatter DATE_FORMATTER
+            = DateTimeFormatter.ofPattern(DATE_PATTERN);
+    private static final int FIRST_ELEMENT = 0;
+    private static final int SECOND_ELEMENT = 1;
```

Comment:
```suggestion
    private static final int TYPE_INDEX = 0;
    private static final int AMOUN_INDEX = 1;
```
NAMES ARE NOT INFORMATIVE